konnectivity-client/pkg/client/client.go

```diff
@@ -281,33 +280,34 @@ func (t *grpcTunnel) serve(tunnelCtx context.Context) {
 			// TODO: flow control
 			conn, ok := t.conns.get(resp.ConnectID)
 
-			if ok {
-				timer := time.NewTimer((time.Duration)(t.readTimeoutSeconds) * time.Second)
-				select {
-				case conn.readCh <- resp.Data:
-					timer.Stop()
-				case <-timer.C:
-					klog.ErrorS(fmt.Errorf("timeout"), "readTimeout has been reached, the grpc connection to the proxy server will be closed", "connectionID", conn.connID, "readTimeoutSeconds", t.readTimeoutSeconds)
-					return
-				case <-tunnelCtx.Done():
-					klog.V(1).InfoS("Tunnel has been closed, the grpc connection to the proxy server will be closed", "connectionID", conn.connID)
-				}
-			} else {
+			if !ok {
 				klog.V(1).InfoS("connection not recognized", "connectionID", resp.ConnectID)
```

Comment:
Since we're making cosmetic changes here, can we start this log line with capitals (consistent with other log lines)